provisioner/aws-manager.js

```diff
@@ -189,10 +173,9 @@ AwsManager.prototype.update = function () {
  * state about which type/region/zone combinations are not working well right
  * now
  */
-AwsManager.prototype.handleStalledRequests = function (spotReqs) {
-  var that = this;
-  return Promise.all(Object.keys(spotReqs).map(function (region) {
-    return that.killCancel(region, [], spotReqs[region].map(function (sr) {
+AwsManager.prototype.handleStalledRequests = async function (spotReqs) {
+  await Promise.all(Object.keys(spotReqs).map(region => {
```

Comment:
Tip: lodash has a `_.map` that works well on objects... I prefer to avoid ever having to iterate on keys in objects, but convert to list of objects... But when you have to `_.map({key: value}, (value, key) => {})` works very well, only slight annoyance is that it flips keys and values in the arguments, but I can live with that, as if the object is a list, the key is the index.